File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    result = 0
    for i in range(0, len(t)):
        if len(t[i:len(p)+i:]) == len(p) and int(t[i:len(p)+i:]) <= int(p):
            result += 1
            
    return result
    

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    let result = 0;
    
    for (let i = 0; i < t.length; i++) {
        if (t.slice(i, i + p.length).length === p.length) {
            if (Number(t.slice(i, i + p.length)) <= Number(p)) {
                result++;
            }
        }
    }
    return result;
}